draftjs_exporter/engines/string.py

```diff
@@ -32,54 +32,53 @@ def escape(s):
 }
 
 
+class Elt(object):
+    def __init__(self, type_, attr, children, markup):
```

Comment:
When creating `Elt` object, it seems you often have to pass `None` for `attr`, `children` and `markup` so I reckon those should be optional arguments.

In addition, `children` use to always be a `list` with the old `dict` implementation but now it might be `None`. Since in some places the code does `elt.children.append(child)`, if `children` is `None`, it will crash.

Proposed implementation:
```python
def __init__(self, type_, attr=None, children=None, markup=None):
    self.type = type_
    self.attr = attr
    self.children = children if children is not None else []
    self.markup = markup
```

Does it make sense?

Reply:
👌 `children` was None before when the element is HTML. This is rather hack-ish – I've added a factory method so it's more contained at least.